apps/election-manager/src/__snapshots__/App.test.tsx.snap

```diff
@@ -1826,7 +1826,7 @@ exports[`tabulating CVRs 1`] = `
             <td
               class="sc-fFubgz jVzFBN"
             >
-              54
+              53
```

Comment:
The previous code erroneously counted votes for either/neither contests where the CVR is malformed and only contains one of the two contests, malformed CVRs like that will now be filtered out, causing the number here to decrease by 1 as there is 1 malformed CVR in the test set that has this contest but not the other.